File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    removeDuplicates(nums) {
        let left = 1;
        let iterator = 1;

        while(iterator < nums.length){
            if(nums[iterator] !== nums[iterator-1]) nums[left++] = nums[iterator];
            iterator++;
        }
        return left;
    }
}

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        have_to_reach = len(nums) -1
        for i in range(len(nums)-2, -1,-1):
            if i+nums[i] >= have_to_reach:
                have_to_reach = i
            

        return have_to_reach == 0

        